examples/rstudio/rstudio_auth

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# rstudio_auth method from https://github.com/nickjer/singularity-rstudio
+
+# Confirm username is supplied
+if [[ $# -ne 1 ]]; then
+  exit 1
+fi
+USERNAME="${1}"
```

Comment:
Does this need to be interactive? Can we use some other method to ensure the caller is part of the test? I'd think we don't actually need to authenticate for the purposes of this test, since we're just passing around dummy passwords stored elsewhere in the test.

Reply:
It doesn't, and as far as I can tell there's no way for the user to interact with the interactive parts. This script is essentially a kludge that gets around having to setup some authentication rstudio requires. I believe the important part is just that it exits with `0` to rstudio. I'll set it to just `exit 0` always and not check the password. The server being on a particular port ensures the caller is part of the test.